Game Dev/Assets/MonsterSpawner.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+using UnityEngine;
+
+public class MonsterSpawner : MonoBehaviour
+{
+    public Transform[] spawnPoints;
+    //public GameObject[] monsters;
+    public GameObject monster;
+    int randomSpawnPoints;
+
+    public static bool spawnAllowed;
+
+    private int birdNumber;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        birdNumber = GameObject.FindGameObjectsWithTag("bird").Length;
```

Comment:
Tag should be provided as parameter/value in editor so we can use the same script to make spawns for all kinds of enemies.